### PR TITLE
Added README.md to help new users get started

### PR DESCRIPTION
**Pull request by glb**
I wasn't quite sure where to get started, so here's what I came up with after poking around a bit.

### CODE REVIEW

**Comment by shehackspurple** [comment]
Thanks Geoff!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+The application is defined in the `docker-compose.yaml` file; to run it,
+make sure you have `docker-compose` installed
+([instructions](https://docs.docker.com/compose/install/)) and then run:
+
+```
+docker-compose up
+```
+
+This will expose the following ports:
```
Comment (kingthorin) [inline README.md:15]: `resources` or `services` might be better than "ports".

The URLs could be proper markdown links.

Comment (glb) [reply README.md:15]: @kingthorin thanks; do you have an example of what you might prefer to see for the URLs? It seems redundant to me to have something like `[http://localhost:8000/](http://localhost:8000)` when common renderers will detect and create the link, so I'm guessing that is not what you meant.

Comment (kingthorin) [reply README.md:15]: You know what, when I was reviewing it i was just seeing the text and not thinking it through. You're right github will render it as a link and it'll be fine. Disregard....sry. :zap:

**Comment by glb** [comment]
@shehackspurple ping? is this PR still relevant / is the content accurate? Would you like changes? Should I just close the PR as stale?

**Comment by shehackspurple** [comment]
Nicole, do you want to talk to Geoff?

Tanya Janca
@SheHacksPurple
https://medium.com/@shehackspurple

DevSlop Live! Sundays 1:00 pm ETD
Mixer <https://aka.ms/DevSlop-Mixer> & Twitch <https://aka.ms/DevSlopTwitch>
http://aka.ms/devslopshow <http://aka.ms/devslopshow>

**Comment by glb** [comment]
Silly me, I forgot that you said this repo had moved to github.com/devslop .. just reviewing my open PRs and didn't see / remember that conversation here.

**Pull request closed without merging**